nifi-nar-bundles/nifi-azure-bundle/nifi-azure-services-api/src/main/java/org/apache/nifi/services/azure/data/explorer/KustoQueryResponse.java

```diff
@@ -27,16 +28,27 @@ public class KustoQueryResponse {
 
     private final String errorMessage;
 
+    private final List<List<Object>> queryResult;
```

Comment:
Now that the interface has been redefined, can this list of results be removed? The values should be passed in the service implementation methods, which would avoid adding this member variable to the public response object.